app/view/pages/github/index.js

```diff
@@ -0,0 +1,95 @@
+import React, {useEffect, useState} from 'react';
+import {Animated} from 'react-native';
+import {useSelector, useDispatch} from 'react-redux';
+import {useNavigation} from 'react-navigation-hooks';
+
+import Item from './Item';
+import {View, FlatList, Loading, Appbar, Button, Text} from 'view/components';
+
+import i18n from 'i18n';
+import style from 'view/style';
+
+import {getRepos} from 'store/reducers/github';
+import {selectGithub} from 'store/selectors/github';
+
+import buttons from './buttons';
+
+const Github = () => {
+  const {appLang, githubLang} = i18n;
+  const {appStyle, githubStyle} = style;
+
+  const [query, setQuery] = useState('react');
+  const [scaleValue] = useState(new Animated.Value(0));
+
+  const dispatch = useDispatch();
+  const {goBack} = useNavigation();
+  const github = useSelector(selectGithub);
+
+  const _goBack = () => goBack();
+
+  useEffect(() => {
+    dispatch(getRepos(query));
+  }, [dispatch, query]);
+
+  useEffect(() => {
+    if (!github.loading && github.data.length) {
+      Animated.timing(scaleValue, {
+        delay: github.data[query] * 350,
+        duration: 600,
+        toValue: 1,
+      }).start();
+    }
+  }, [github, query, scaleValue]);
+
+  return (
+    <View style={appStyle.container}>
+      <Appbar.Header style={appStyle.header}>
+        <Appbar.Action onPress={() => _goBack()} icon="arrow-back" />
+        <Appbar.Content title={githubLang.title} />
+      </Appbar.Header>
+
+      <View style={appStyle.content}>
+        <View style={githubStyle.buttonArea}>
+          {buttons.map(button => (
+            <Button
+              key={button.id}
+              mode="outlined"
+              color={'#13a77f'}
+              disabled={github.loading}
+              onPress={() => setQuery(button.id)}
+              style={appStyle.button}>
+              {github.loading
```

Comment:
Using ternary inside another ternary works well but causes to bad readability. Think hard to write it better. I have no idea about it.

Reply:
it's in a better shape now

Reply:
What do your words mean? Do you mean you will change it to a better case? Or you are serious about using ternary inside ternary?

Reply:
I already changed it 
```
{query === button.id && github.loading
                ? appLang.loading
                : button.title}
```